portsyncd/linksync.cpp

```diff
@@ -151,6 +154,8 @@ LinkSync::LinkSync(DBConnector *appl_db, DBConnector *state_db) :
             }
         }
     }
+
+    if_freenameindex(if_ni);
```

Comment:
>if_freenameindex [](http://example.com/codeflow?start=4&length=16)

Some exception may be thrown in the middle of this function. It may skip the free statement.

Reply:
Fixed in https://github.com/Azure/sonic-swss/pull/2221/commits/9a2175befd44a6813f5b9fc0b3401fc4498c1419